docs/src/installation.md

```diff
@@ -115,6 +115,7 @@ The link in the `Solver` column is the corresponding Julia package.
 | [CPLEX](https://www.ibm.com/analytics/cplex-optimizer/)                        | [CPLEX.jl](https://github.com/jump-dev/CPLEX.jl)                                 | Manual | Comm.    | (MI)LP, (MI)SOCP          |
 | [CSDP](https://github.com/coin-or/Csdp)                                        | [CSDP.jl](https://github.com/jump-dev/CSDP.jl)                                   |        | EPL      | LP, SDP                   |
 | [DAQP](https://github.com/darnstrom/daqp)                                      | [DAQP.jl](https://github.com/darnstrom/DAQP.jl)                                  |        | MIT      | (Mixed-binary) QP         |
+| [DSDP](http://www.mcs.anl.gov/hs/software/DSDP/)                               | [DSDP.jl](https://github.com/jump-dev/DSDP.jl)                                   |        | ???      | LP, SDP                   |
```

Comment:
```suggestion
| [DSDP](http://www.mcs.anl.gov/hs/software/DSDP/)                               | [DSDP.jl](https://github.com/jump-dev/DSDP.jl)                                   |        | DSDP     | LP, SDP                   |
```

This seems to be a very unique license, but it's in spdx as "DSDP" so we should go with that:

 * https://spdx.org/licenses/DSDP.html
 * https://fedoraproject.org/wiki/Licensing/DSDP